e2e/specs/st_file_uploader.spec.js

```diff
@@ -170,13 +178,22 @@ describe("st.file_uploader", () => {
             force: true,
             subjectType: "drag-n-drop",
             events: ["dragenter", "drop"]
-          })
+          });
+
+        cy.get(".uploadedFileName").each(uploadedFileName => {
+          cy.get(uploadedFileName).should("have.text", fileName1);
+        });
+
+        cy.get("[data-testid='stFileUploadDropzone']")
+          .eq(1)
           .attachFile(files[1], {
             force: true,
             subjectType: "drag-n-drop",
             events: ["dragenter", "drop"]
           });
 
+        cy.wait("@uploadFile");
```

Comment:
Hm, I think I'm missing something -- why are we calling cy.wait after the second file is uploaded, as opposed to before? 

(In the previous version of the code, cy.wait(500) was before the second upload, which made more intuitive sense to me. But maybe the goal of the wait is different this time?)

Reply:
Before we were calling a random wait but now we moved the assertion to check if one file has been uploaded. This in itself comes with a built in wait for the uploaded file to become available. However, since this element is already available, when we do another check to see what is there, it doesn't wait for the request to finish so here we are waiting for the HTTP request to finish. Tried to check for the progress bar showing up but at least on my machine, the file gets uploaded too quickly for the progress bar to get rendered. Also waiting for the request to finish seems more reasonable

Reply:
Ah, makes sense! Thanks for the explanation. Could we capture this briefly as code comments?